Cubical/HITs/Truncation/Base.agda

```diff
@@ -1,29 +1,32 @@
+{-
+
+A simpler definition of truncation ∥ A ∥ n from n ≥ -1
+
+Note that this uses the HoTT book's indexing, so it will be off
+ from `∥_∥_` in HITs.Truncation.Base by -2
+
+-}
 {-# OPTIONS --cubical --no-import-sorts --safe #-}
 module Cubical.HITs.Truncation.Base where
 
+open import Cubical.Data.NatMinusOne renaming (suc₋₁ to suc)
 open import Cubical.Foundations.Prelude
 open import Cubical.Foundations.HLevels
-open import Cubical.Data.Nat
-open import Cubical.Data.NatMinusOne
-open import Cubical.HITs.Nullification
 open import Cubical.HITs.Sn
+open import Cubical.Data.Nat.Base renaming (suc to sucℕ)
+open import Cubical.Data.Unit.Base
+open import Cubical.Data.Empty
 
--- For the hub-and-spoke construction discussed in the HoTT book, which doesn't work in the base case
---  of contractibility, see `HITs.Truncation.FromNegOne`. The definition of truncation here contains
---  two more constructors which are redundant when n ≥ 1 but give contractibility when n = 0.
-
--- data hLevelTrunc {ℓ} (n : HLevel) (A : Type ℓ) : Type (ℓ-max ℓ ℓ') where
---   -- the hub-and-spoke definition in `Truncation.FromNegOne`
---   ∣_∣ : A → hLevelTrunc n A
---   hub   : (f : S (-1+ n) → hLevelTrunc n A) → hLevelTrunc n A
---   spoke : (f : S (-1+ n) → hLevelTrunc n A) (s : S) → hub f ≡ f s
---   -- two additional constructors needed to ensure that hLevelTrunc 0 A is contractible
---   ≡hub   : ∀ {x y} (p : S (-1+ n) → x ≡ y) → x ≡ y
---   ≡spoke : ∀ {x y} (p : S (-1+ n) → x ≡ y) (s : S (-1+ n)) → ≡hub p ≡ p s
+-- this definition is off by one. Use hLevelTrunc or ∥_∥ for truncations
+-- (off by 2 w.r.t. the HoTT-book)
+data HubAndSpoke {ℓ} (A : Type ℓ) (n : ℕ) : Type ℓ where
```

Comment:
Please change N to HLevel here

Reply:
This might be confusing since this is not the HLevel, but HLevel - 1

Reply:
Oh, right! Can you add comment about it